packages/aws-cdk/lib/commands/migrate.ts

```diff
@@ -0,0 +1,71 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { initializeProject, availableInitTemplates } from '../../lib/init';
+import { warning } from '../logging';
+import * as nocti from '../vendor/noctilucent';
+
+/** The list of languages supported by the built-in noctilucent binary. */
+export const MIGRATE_SUPPORTED_LANGUAGES: readonly string[] = nocti.supported_languages();
+
+export async function cliMigrate(
+  inputpath: string = process.cwd() + '/../temp.txt',
+  language = MIGRATE_SUPPORTED_LANGUAGES[0],
+  generateOnly = false,
+  outputpath = process.cwd(),
+) {
+  warning('This is a very experimental feature, use at your own risk.');
+  const type = 'default'; // "default" is the default type (and maps to 'app')
```

Comment:
I had to double-take: why is this an option here, but not exposed as an argument?
Even if right now it has a single value, it can be expanded in the future (based on the comment)